client/src/entrypoints/admin/page-editor.js

```diff
@@ -61,6 +61,16 @@ function initErrorDetection() {
       .addClass('!w-flex')
       .find('[data-tabs-errors-count]')
       .text(errorSections[index]);
+
+    const ErrorStatement = $('[data-tabs] a[href="#' + index + '"]')
+      .find('[data-tabs-errors]')
+      .find('[data-tabs-errors-statement]');
+
+    if (errorSections[index] === 1) {
+      ErrorStatement.text(`${errorSections[index]} error`);
+    } else if (errorSections[index] > 1) {
+      ErrorStatement.text(`${errorSections[index]} errors`);
+    }
```

Comment:
Firstly, a small improvement while we are here in this file...

* Line 45 - let's get rid of the `// eslint-disable-next-line func-names` and give this a function name
* `$('.error-message,.help-critical').each(function collectError() {`

Then, we should revisit the next block of code (line 56 through to 73)

* Let's avoid the for index in errorSections and the linting ignore and instead use [`Object.entries`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries) to go through each section.
* Then, we should try to avoid 're-getting' the DOM element via the `data-tabs` traversal and pull this div container out to a variable.
* Also, when giving variables to elements, it's best to use lowerKebabCase to indicate that this is not a class, sometimes it's good to use `$lowerKebabCase` to indicate it's a jQuery instance (but we do not use this consistently so all good).
* Then, when it comes to injecting the text for translations and pluralisation we should never ever use bare text as is, this will be impossible for translators to translate.
* Instead, in JavaScript we have the `gettext` utils available that are essentially Django's JS translation utils -> https://docs.djangoproject.com/en/4.1/topics/i18n/translation/#internationalization-in-javascript-code
* We also need to wrap the error count in brackets as suggested in the issue. This way we get 'tab name (3 errors)' not 'tab name 3 errors'. This reads better for these simple tab names.


```javascript
function initErrorDetection() {
  const errorSections = {};

  // first count up all the errors
  $('.error-message,.help-critical').each(function collectError() {
    const parentSection = $(this).closest('section[role="tabpanel"]');

    if (!errorSections[parentSection.attr('id')]) {
      errorSections[parentSection.attr('id')] = 0;
    }

    errorSections[parentSection.attr('id')] =
      errorSections[parentSection.attr('id')] + 1;
  });

  // now identify them on each tab
  Object.entries(errorSections).forEach(([sectionId, errorCount]) => {
    const tabErrorsElement = $(`[data-tabs] a[href="#${sectionId}"]`).find(
      '[data-tabs-errors]',
    );

    // update label for screen readers
    tabErrorsElement
      .find('[data-tabs-errors-statement]')
      .text(
        ngettext(
          '({errorCount} error)',
          '({errorCount} errors)',
          errorCount,
        ).replace('{errorCount}', errorCount),
      );

    // show and add error count
    tabErrorsElement
      .addClass('!w-flex')
      .find('[data-tabs-errors-count]')
      .text(errorCount);
  });
}
```

This will change the output to the following content.

<img width="1848" alt="Screen Shot 2022-11-20 at 8 07 30 pm" src="https://user-images.githubusercontent.com/1396140/202896554-d5c71fc6-aed0-49da-ae97-32009b828338.png">